pkgs/development/python-modules/mat2/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, buildPythonPackage
+, python
+, pythonOlder
+, fetchFromGitLab
+, substituteAll
+, bubblewrap
+, exiftool
+, ffmpeg
+, mime-types
+, wrapGAppsHook
+, gdk-pixbuf
+, gobject-introspection
+, librsvg
+, poppler_gi
+, mutagen
+, pygobject3
+, pycairo
+}:
```

Comment:
This argument list should be sorted.

Reply:
It would be nice but is not strictly required.

Reply:
They are more or less ordered by appearance inthe expression.